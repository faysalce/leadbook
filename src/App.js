import React from 'react';
import ErrorBoundary from "./components/ErrorBoundary"
import PageLayout from './components/PageLayout'
function App() {
  return (
    <ErrorBoundary>
      <PageLayout/>
    </ErrorBoundary>
    
  );
}

export default App;
