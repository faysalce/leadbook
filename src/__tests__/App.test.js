import React from 'react';
import App from '../App';
import {shallow} from "enzyme";

it("App renders correctly", () => {
  const wrapper = shallow(
    <App />
  );
  expect(wrapper).toMatchSnapshot();
});