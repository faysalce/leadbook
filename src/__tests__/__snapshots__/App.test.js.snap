// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `ShallowWrapper {}`;

exports[`renders correctly 1`] = `ShallowWrapper {}`;
